Makefile

```
@@ -197,7 +213,8 @@ ifneq ($(USE_LIST_SO),1)
 endif
 SOURCES  := $(SRC_SOURCES) $(LIB_SOURCES)
 
-GENERAL_SOURCES := $(shell find $(SRCDIR)/utils -name "*.c") $(shell find $(SRCDIR)/account -name "*.c") $(shell find $(SRCDIR)/ipc -name "*.c") $(shell find $(SRCDIR)/defines -name "*.c")
+GENERAL_SOURCES := $(shell find $(SRCDIR)/utils -name "*.c") $(shell find $(SRCDIR)/account -name "*.c") $(shell find $(SRCDIR)/ipc -name "*.c") $(shell find $(SRCDIR)/defines -name "*.c") $(shell find $(SRCDIR)/api -name "*.c")
+GENERAL_SOURCES := $(filter-out $(SRCDIR)/ipc/windows/connection.c $(SRCDIR)/ipc/windows/cryptCommunicator.c $(SRCDIR)/ipc/windows/cryptIpc.c $(SRCDIR)/ipc/windows/ipc.c, $(GENERAL_SOURCES))
```

If this works, this is better
```suggestion
GENERAL_SOURCES := $(filter-out $(SRCDIR)/ipc/windows/*.c, $(GENERAL_SOURCES))
```

---

unfortunately, this didn't work out :( So I decided to keep a simple list